app/views/rubric_criteria/_rubric_criterion_levels.html.erb

```diff
@@ -1,5 +1,5 @@
 <div id='rubric_levels_top'></div>
-<% (0...RubricCriterion::RUBRIC_LEVELS).each do |rubric_level_index| %>
+<% (0...criterion.levels.length).each do |rubric_level_index| %>
```

Comment:
This is good, but you can do even better by doing `criterion.levels.each_with_index ...` (look up this method)